mitodl/open-discussions
# Mb/stafflist items serializer

#### Pre-Flight checklist


#### What are the relevant tickets?
Related to #3871 

#### What's this PR do?
Adds a `stafflists` property to learning resource objects indicating which stafflists they belong to.

#### How should this be manually tested?
Create some stafflists and add the same course to each of them (see PR # for details on how to do so).
Go to `../api/v0/courses/<course_id>` and check that there is a `stafflists` property with a list of items, one for each stafflist the course is in (assuming you are logged in as admin or a stafflist editor.  If not, the list should be empty).
